controllers/iperf3/serverdeployment.go

```diff
@@ -49,13 +55,20 @@ func (r *Reconciler) newServerDeployment(ctx context.Context, cr *perfv1alpha1.I
 		labels[k] = v
 	}
 
-	iperfCmdLineArgs := fmt.Sprintf("-s -p %s %s",
-		strconv.Itoa(iperf3ServerPort),
-		cr.Spec.ClientConfiguration.CmdLineArgs)
+	iperfCmdLineArgs := fmt.Sprintf("--server --port %d ",
```

Comment:
I prefer using arrays and `strings.Join` instead of appending strings with whitespace. `%d` was another suggestion :)
```golang
var cmds = []string{fmt.Sprintf("--server --port %d ", Iperf3ServerPort)}
if cr.Spec.UDP {
    cmds.append(cmds, "--udp")
}
cmds.append(cr.Spec.ClientConfiguration.CmdLineArgs)
// and use strings.Join(cmds, " ")
```

Reply:
done